### ISSUE TITLE
Detailled Description not shown but got stored successfully

### ISSUE THREAD

**Issue by Dreifels** [Nexus-Mods/web-issues#835]
## Bug/error being reported:
Detailled Description shown when I enter "manage" but not public, althought saved several times 

## Steps to reproduce:
https://www.nexusmods.com/skyrimspecialedition/mods/20317
no details to see, only "about" one line.

If I login and goto "manage", I see all (picture , text I wrote). So it is stored, but not published.

**Comment by Pickysaurus** [comment]
Is this still happening? Looks alright to me. 
Do you have a screenshot of it displaying incorrectly?

**Comment by Dreifels** [comment]
It looked like the image on details page got not shown as image but its code, 
the image on details page I placed by pull over from another Nexs page.
Then I did save the image local and copy-past it to the details page, and after that the page was fine after publishing. 
Not yet tested if this will still happen. The pages I published later I always took the image fromlocal stored one, nt by pull over from a Nexus publication. 
However, before that happened, I always did work by that way and it was fine.
